docs/install/any-kubernetes-cluster.md

```diff
@@ -682,6 +686,10 @@ At this point, you have a basic installation of Knative Eventing!
 
 ### Optional Eventing extensions
 
+You can apply extensions to customize your Knative Eventing installation.
+For more information about the extensions you can apply, see
+[Installation files](./custom-install/custom-install.md#knative-eventing-installation-files).
```

Comment:
Sorry, one more thing - these links need to be updated to the new filename 🙂

Reply:
Addressed!